### DISCUSSION TITLE
Self-nomination for Contributor role

### DISCUSSION THREAD

**Discussion by joshuatcasey** [paketo-buildpacks/feedback#147] [Ruby Team]
Hi @paketo-buildpacks/ruby-maintainers, I am nominating myself as a ruby contributor. I've helped out in a few ways such as the recent dependency work.

Thanks, Josh

**Comment by robdimsdale** [comment]
A big 👍 from me - thanks for all the work on adding support for the dependency work to the ruby ecosystem!
